test/remote/gateways/remote_shift4_test.rb

```diff
@@ -0,0 +1,249 @@
+require 'test_helper'
+
+class RemoteShift4Test < Test::Unit::TestCase
+  def setup
+    @gateway = Shift4Gateway.new(fixtures(:shift4))
+    @access_token = @gateway.options[:access_token]
```

Comment:
No need to get them as we will already be passing it through fixtures, after updating the code changes as requested above: https://github.com/activemerchant/active_merchant/pull/4415/files#r897877558